docs/setup-guide.md

```diff
@@ -12,101 +12,95 @@
 
 1. Install Salesforce CLI. You can install the CLI with either npm or with a downloadable installer for your specific operating system. See [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm) for more information.
     ```bash
-    npm install --global sfdx-cli
+    npm install @salesforce/cli@2.43.7 --global
```

Comment:
They don't have to enter a version name.

Reply:
The import functionality was pushed as a part of 2.43.7 itself and as the pipelines try to install 2.42.x. the new SF import command fails, Hence added with the specific version.

Will  remove this before moving this to main/release.